webknossos/webknossos/dataset/_utils/buffered_slice_writer.py

```diff
@@ -34,6 +34,7 @@ def __init__(
         self,
         view: "View",
         offset: Optional[Vec3IntLike] = None,
+        update_bbox: bool = False,
```

Comment:
I’d say the default should be `True`, and there could be a comment explaining what it does and to encourage setting it to false IF parallel access is intended.